azurerm/internal/services/blueprints/blueprint_definition_datasource.go

```diff
@@ -0,0 +1,132 @@
+package blueprints
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/blueprints/validate"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/timeouts"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmBlueprintDefinition() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmBlueprintDefinitionRead,
+
+		Timeouts: &schema.ResourceTimeout{
+			Read: schema.DefaultTimeout(5 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validate.BlueprintName,
+			},
+
+			"scope_type": {
+				Type:     schema.TypeString,
+				Required: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"subscriptions",
+					"managementGroup",
+				}, false),
+			},
+
+			"scope_name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringIsNotEmpty,
+			},
```

Comment:
Revisiting this with fresh eyes after a break, I think this does need simplifying.  Since this sets the target scope, which cannot be a `managementGroup`, I'll re-work this to being a more appropriate specific item, perhaps `target_subscription_id`?